easybuild/framework/easyblock.py

```diff
@@ -2943,20 +2943,42 @@ def run_all_steps(self, run_test_cases):
 
         print_msg("building and installing %s..." % self.full_mod_name, log=self.log, silent=self.silent)
         trace_msg("installation prefix: %s" % self.installdir)
-        try:
-            for (step_name, descr, step_methods, skippable) in steps:
-                if self._skip_step(step_name, skippable):
-                    print_msg("%s [skipped]" % descr, log=self.log, silent=self.silent)
-                else:
-                    if self.dry_run:
-                        self.dry_run_msg("%s... [DRY RUN]\n", descr)
-                    else:
-                        print_msg("%s..." % descr, log=self.log, silent=self.silent)
-                    self.current_step = step_name
-                    self.run_step(step_name, step_methods)
 
-        except StopException:
-            pass
+        lockpath = build_option('lockpath') or os.path.join(install_path('software'), '.locks')
+        if not os.path.exists(lockpath):
+            mkdir(lockpath)
```

Comment:
@mboisson Please change this to `mkdir(lockpath, parents=True)` (equivalent with `mkdir -p`)

Reply:
done